contracts/StaderConfig.sol

```diff
@@ -297,6 +297,27 @@ contract StaderConfig is IStaderConfig, AccessControlUpgradeable {
         setContract(SD_INCENTIVE_CONTROLLER, _sdIncentiveController);
     }
 
+    // Access Control
+    function giveCallPermission(
```

Comment:
one concern about this approach is, if we upgrade the functionSig, we have to take care that we revoke permission from previous functionSig and grant permission to new functionSig. Hence this would have to be added as part of upgrade txns. Hence a little extra overhead. And hopefully people shouldn't miss this step.

Reply:
[27fc0bc](https://github.com/stader-labs/ethx/pull/253/commits/27fc0bc3dbf0b42bb598422518b4e73ea9702530) legacy mechanism used for access control